src/services/user/model.js

```diff
@@ -56,6 +58,10 @@ const userSchema = new Schema({
 	timestamps: true,
 });
 
+userSchema.index({ schoolId: 1, roles: -1 });
```

Comment:
set school in all schemas, using userGroupSchema instead?

Reply:
will Monitor this, for now I think more specific is better